legate/pyproject.toml

```diff
@@ -0,0 +1,118 @@
+# Copyright (c) 2021-2022, NVIDIA CORPORATION. All rights reserved.
+# See file LICENSE for terms.
+
+[build-system]
+build-backend = "setuptools.build_meta"
+requires = [
+    "cmake>=3.23.1,!=3.25.0",
+    "cython>=0.29,<0.30",
+    "ninja",
+    "scikit-build>=0.13.1",
+    "setuptools",
+    "wheel",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../dependencies.yaml and run `rapids-dependency-file-generator`.
+
+[project]
+name = "legate_kvikio"
+version = "23.08.00"
+description = "KvikIO - GPUDirect Storage"
+readme = { file = "README.md", content-type = "text/markdown" }
+authors = [
+    { name = "NVIDIA Corporation" },
+]
+license = { text = "Apache 2.0" }
+requires-python = ">=3.9"
+dependencies = [
+    "cupy-cuda11x>=12.0.0",
+    "numpy>=1.21",
+    "zarr",
+] # This list was generated by `rapids-dependency-file-generator`. To make changes, edit ../dependencies.yaml and run `rapids-dependency-file-generator`.
+classifiers = [
+    "Intended Audience :: Developers",
+    "Topic :: Database",
+    "Topic :: Scientific/Engineering",
+    "License :: OSI Approved :: Apache Software License",
+    "Programming Language :: Python",
+    "Programming Language :: Python :: 3.9",
+    "Programming Language :: Python :: 3.10",
```

Comment:
I am wondering if it is best just to remove the `Python ::<version>` lines? Are they used by anyone?